dts/arm/nordic/nrf54l15_cpuapp.dtsi

```diff
@@ -37,6 +36,27 @@ cpuflpr_vevif: &cpuflpr_vevif_remote {};
 	status = "okay";
 };
 
+&cpuflpr_vpr {
+	cpuapp_vevif_rx: mailbox@1 {
+		compatible = "nordic,nrf-vevif-event-rx";
+		reg = <0x0 0x1000>;
```

Comment:
```
unit address and first address in 'reg' (0x5004c000) don't match for /soc/peripheral@50000000/vpr@4c000/mailbox@1
```

@jaz1-nordic I think this should be `reg = <0x1 0x1000>;`?

Reply:
cc @gmarull @anangl

Reply:
needs fixing @jaz1-nordic